.chloggen/release-ack-extension.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: ackextension
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "releasing the ackextension in alpha stability"
```

```suggestion
note: "Promote the to `alpha` stability"
```
Just a nit, formatted it to be a bit more similar to other release notes of the same kind.